.travis.yml

```diff
@@ -1,5 +1,7 @@
-sudo: false
+sudo: true
 language: ruby
+before_install:
+  - sudo apt-get install -qq augeas-tools augeas-lenses libaugeas-dev
```

Comment:
Unfortunately I think a newer version than precise ships is required, unless ruby-augeas is pinned to an older version, so it probably excludes using this.